package.json

```diff
@@ -30,13 +30,13 @@
     "prettier": "2.8.8"
   },
   "optionalDependencies": {
-    "@sentry/cli-darwin": "2.31.2",
-    "@sentry/cli-linux-arm": "2.31.2",
-    "@sentry/cli-linux-arm64": "2.31.2",
-    "@sentry/cli-linux-i686": "2.31.2",
-    "@sentry/cli-linux-x64": "2.31.2",
-    "@sentry/cli-win32-i686": "2.31.2",
-    "@sentry/cli-win32-x64": "2.31.2"
+    "@sentry/cli-darwin": "2.31.0",
+    "@sentry/cli-linux-arm": "2.31.0",
+    "@sentry/cli-linux-arm64": "2.31.0",
+    "@sentry/cli-linux-i686": "2.31.0",
+    "@sentry/cli-linux-x64": "2.31.0",
+    "@sentry/cli-win32-i686": "2.31.0",
+    "@sentry/cli-win32-x64": "2.31.0"
   },
```

Comment:
Can you please rebase on master? We should not be reducing the version number here (or changing this file in any way)